### PR TITLE
Adds LOOC to runechat.

### PR DESCRIPTION
**Pull request by lewcc**
## What Does This PR Do
Adds LOOC to runechat. If you have runechat enabled in your preferences, you'll now see in-game if someone's LOOCing you. This will be visible to anyone around you, including ghosts, even if they cannot hear you. 
Note that ghosts should be able to see LOOC runechat, but their own LOOC messages will not be runechatted. 

## Why It's Good For The Game
LOOC can be easy to miss if a) you have a bunch of logs running through your chatbox, b) you're in a really busy situation, or c) you never really check chat anyway. Since LOOC messages tend to be pretty useful, especially for getting the attention of newbies who might not be used to reaching chat anyway, this seems like it'd be a net benefit.

Plus, it was also requested by a few folks (thanks tourte for the idea).

## Images of changes
image

## Testing
Loaded in, tried LOOCing and saw it runechatted.

## Changelog
:cl:
add: LOOC is now visible in runechat, if you have it enabled.
/:cl:

### CODE REVIEW

File: code/game/verbs/ooc.dm [hunk 1]
```diff
@@ -209,6 +209,9 @@ GLOBAL_VAR_INIT(admin_ooc_colour, "#b82e00")
 
 	log_looc(msg, src)
 	mob.create_log(LOOC_LOG, msg)
+	for(var/mob/M in get_mobs_in_view(7, mob))
+		if((M.client?.prefs.toggles2 & PREFTOGGLE_2_RUNECHAT) && M.can_hear())
```
Comment (larentoun) [inline code/game/verbs/ooc.dm:213]: If it's LOOC, shoudn't it use `viewers()` or something instead of `can_hear()`? Deaf people should see LOOC

Comment (lewcc) [reply code/game/verbs/ooc.dm:213]: You're right, that's a copy paste mistake on my part

File: code/game/verbs/ooc.dm [hunk 2]
```diff
@@ -209,6 +209,10 @@ GLOBAL_VAR_INIT(admin_ooc_colour, "#b82e00")
 
 	log_looc(msg, src)
 	mob.create_log(LOOC_LOG, msg)
+	if(!isobserver(mob))
```
Comment (Contrabang) [inline code/game/verbs/ooc.dm:212]:
```suggestion
	if(isliving(mob))
```

**Review by Contrabang — CHANGES_REQUESTED** [review]
Dont really want to see this stuff from camera mobs probably

**Review by DGamerL — APPROVED** [review]

File: code/datums/chatmessage.dm [hunk 3]
```diff
@@ -133,6 +133,10 @@
 		if(RUNECHAT_SYMBOL_EMOTE)
 			symbol = "<span style='font-size: 9px; color: #3399FF;'>*</span> "
 			size = size || "small"
+		if(RUNECHAT_SYMBOL_LOOC)
+			symbol = "<span style='font-size: 5px; color: #6699cc;'><b>\[LOOC]</b></span> "
+			size = size || "small"
```
Comment (Contrabang) [inline code/datums/chatmessage.dm:138]:
```suggestion
			size ||= "small"
```

File: code/datums/chatmessage.dm [hunk 4]
```diff
@@ -133,6 +133,10 @@
 		if(RUNECHAT_SYMBOL_EMOTE)
 			symbol = "<span style='font-size: 9px; color: #3399FF;'>*</span> "
 			size = size || "small"
```
Comment (Contrabang) [inline code/datums/chatmessage.dm:135]:
```suggestion
			size ||= "small"
```

**Review by Contrabang — COMMENTED** [review]
minor stuff, i think its clearer to use the `||=` operator here (my favorite operator), up to you really. LGTM tho

**Review by Contrabang — APPROVED** [review]

**Review by S34NW — APPROVED** [review]

**Pull request merged**